### PR TITLE
Custom Executor Example

### PR DESCRIPTION
**Pull request by domenukk**

### CODE REVIEW

File: fuzzers/baby/baby_fuzzer_custom_executor/src/main.rs [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+    // Feedback to rate the interestingness of an input
+    let mut feedback = MaxMapFeedback::new(&observer);
+
+    // A feedback to choose if an input is a solution or not
+    let mut objective = feedback_and_fast!(
+        // Look for crashes.
+        CrashFeedback::new(),
+        // We `and` the MaxMapFeedback to only end up with crashes that trigger new coverage.
+        // We use the _fast variant to make sure it's not evaluated every time, even if the crash didn't trigger..
+        MaxMapFeedback::new(&observer)
```
Comment (domenukk) [inline fuzzers/baby/baby_fuzzer_custom_executor/src/main.rs:103]: How does this never trigger? Seems like a bug - or am I stupid?
Like, if I leave this guy out I get all crashes, if I have it I don't see single one.
I would assume I should get _exactly one_?

cc @tokatoka maybe?

Comment (domenukk) [reply fuzzers/baby/baby_fuzzer_custom_executor/src/main.rs]: Name collision, ... should we add an atomic counter to names?

Comment (tokatoka) [reply fuzzers/baby/baby_fuzzer_custom_executor/src/main.rs]: yeah we should

Comment (domenukk) [reply fuzzers/baby/baby_fuzzer_custom_executor/src/main.rs]: Someone(tm) should do something(tm)

**Pull request merged**